src/templates/app/user/security/self_exclusion.jsx

```diff
@@ -6,7 +6,8 @@ const SelfExclusion = () => (
     <React.Fragment>
         <div className='invisible' id='description'>
             <h1>{it.L('Self-Exclusion Facilities')}</h1>
-            <p>{it.L('Online trading can become addictive. Fill in the form below to limit your participation on the website or send a signed letter or fax to our customer support team. Once set, you can only tighten your limits. Limits will only be removed or loosened after 7 days with the exception of the self-exclusion date, which cannot be removed or altered once you have confirmed it. To remove or increase your limits, please contact [_1]customer support[_2].', `<a href="${it.url_for('contact')}">`, '</a>')}</p>
+            <p>{it.L('Online trading can be addictive. Self-exclusion is a facility to limit your online trading activity, should you need it. You can set limits in your account to help prevent unwanted losses. You may adjust these limits any time, but remove them only after 24 hours.')}</p>
+            <p>{it.L(`Contact us to remove or adjust your limits. [_1] You may also instruct us to exclude you from trading on [_2]${it.website_name}[_3] for a specific period of time. This self-exclusion date cannot be amended once it is set.`, `<a href="${it.url_for('contact')}">https://www.binary.com/en/contact.html</a>`, '<a href="https://binary.com">', '</a>')}</p>
```

Comment:
`[_2]${it.website_name}[_3]` should just be `[_2]` and replace it with the link and it.website_name in the placeholder.